.github/workflows/test-cf-worker-next.yml

```diff
@@ -0,0 +1,73 @@
+name: "Test edge function compatibility for Next.js on Cloudflare Workers"
+on: push
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        env:
+            CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+            CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+            APP_URL: supertokens-node-b95.pages.dev
+            TEST_DEPLOYED_VERSION: true
+        defaults:
+            run:
+                working-directory: examples/next/with-emailpassword
+        steps:
+            - uses: actions/checkout@v2
+            - run: echo $GITHUB_SHA
+            - run: npm install git+https://github.com:supertokens/supertokens-node.git#$GITHUB_SHA
+            - run: npm install
+            - run: npm install mocha@6.1.4 jsdom-global@3.0.2 puppeteer@^11.0.0 isomorphic-fetch@^3.0.0
+
+            # Step to update the runtime to edge to all files in app/api/ and remove pages path
+            # to avoid conflicts
+            - name: Add runtime export to API files
+              run: |
+                  rm -rf pages
+                  find app/api -type f \( -name "*.js" -o -name "*.ts" \) -exec sed -i '1s/^/export const runtime = "edge";\n/' {} +
+                  echo 'export const runtime = "edge";' >> app/auth/[[...path]]/page.tsx
+
+            # Install next on pages to build the app
+            - name: Install next-on-pages
+              run: npm install --save-dev @cloudflare/next-on-pages
+
+            # Setup the compatibility flag to make non edge functions run
+            - name: Create a wrangler.toml
+              run: echo "compatibility_flags = [ "nodejs_compat" ]" >> wrangler.toml
```

Comment:
I think I'm missing something, why do we need this?

Reply:
This is to run the non edge code of the app. `edge` runtime works on non-static routes but static routes still use it(?). I initially tested without the compatibility flags but cloudflare was throwing an error (build would work fine but CF functions would return error) when trying to access the page.

After I added this, it started working fine. However, I have verified that all the non-static code runs on edge runtime. This can be verified by looking at the build logs from the workflow.